src/util/mpir_pmi.c

```diff
@@ -20,6 +20,7 @@ static int pmi_max_val_size;
 #ifdef USE_PMI1_API
 static int pmi_max_kvs_name_length;
 static char *pmi_kvs_name;
+static char *pmi_parent_port;
```

Comment:
The "port" is not intrinsic to `PMI`. It is defined and set in `MPI_Comm_spawn`. That is to say, the mechanism, format, or even the existence of "port", is a MPI implementation detail, rather than a PMI interface.

Can we move "parent_port" to `MPIR_Process`? It shouldn't affect the code much, but the interface separation makes more sense.